### PR TITLE
MOYO header

### PR DESCRIPTION
**Pull request by SergeyShapovalenko**
[DEMO LINK](https://SergeyShapovalenko.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://SergeyShapovalenko.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,13 +2,53 @@
...
+      </li>
+      <li>
+        <a href="#smartphones" class="top-link">Smartphones</a>
+      </li>
+      <li>
+        <a 
+          href="#laptops_computers" 
+          data-qa="hover" 
+          class="top-link">Laptops & Computers
+        </a>
```
Comment (SanyaBratashchuk) [inline src/index.html:38]:
```suggestion
        <a 
          href="#laptops_computers" 
          data-qa="hover" 
          class="top-link"
        >
            Laptops & Computers
        </a>
```

File: src/index.html [hunk 2]
```diff
@@ -2,13 +2,53 @@
...
+      </li>
+      <li>
+        <a href="#photo" class="top-link">Photo</a>
+      </li>
+      <li>
+        <a href="#video" class="top-link">Video</a>
+      </li>
+    </nav>
+  </header>
+</html>
```
Comment (SanyaBratashchuk) [inline src/index.html:54]:
```suggestion
</html>    

```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:3]: Do not apply any rules to the `*` selector except `box-sizing`

File: src/style.css [hunk 4]
```diff
@@ -1 +1,75 @@
...
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px 0;
+}
+
+.top-navigation {
+  display: flex;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:27]: move font-family to the `hmtl` tag selector

File: src/index.html [hunk 5]
```diff
@@ -2,13 +2,53 @@
...
+  <header class="top-menu">
+    <a href="#home">
+      <img 
+        src="images/logo.png" 
+        alt="moyo-logo" 
+        class="logo">
+    </a>
+    <nav class="top-navigation">
+      <li>
+        <a href="#apple" class="is-active">Apple</a>
```
Comment (SanyaBratashchuk) [inline src/index.html:25]:
```suggestion
        <a href="#apple" class="top-link is-active">Apple</a>
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by agniya-i — APPROVED** [review]

**Pull request closed without merging**